astropy/io/ascii/core.py

```diff
@@ -719,13 +734,30 @@ def _set_col_formats(self):
 
 
 def convert_numpy(numpy_type):
-    """Return a tuple ``(converter_func, converter_type)``.  The converter
-    function converts a list into a numpy array of the given ``numpy_type``.
-    This type must be a valid `numpy type
-    <http://docs.scipy.org/doc/numpy/user/basics.types.html>`_, e.g.
-    numpy.int, numpy.uint, numpy.int8, numpy.int64, numpy.float, numpy.float64,
-    numpy.str.  The converter type is used to track the generic data type (int,
-    float, str) that is produced by the converter function.
+    """Return a tuple containing a function which converts a list into a numpy
+    array and the type produced by the converter function.
+    
+    Parameters
+    ----------
+    numpy_type : numpy data-type
+        The numpy type required of an array returned by `converter`. Must be a
+        valid numpy type e.g. numpy.int, numpy.uint, numpy.int8, numpy.int64,
+        numpy.float, numpy.float64, numpy.str.
+        See <http://docs.scipy.org/doc/numpy/user/basics.types.html>
```

Comment:
Links in reST use a different syntax.  See: http://sphinx-doc.org/rest.html#external-links, e.g.:

```
`numpy type <http://docs.scipy.org/doc/numpy/user/basics.types.html>`__
```